README.md

```diff
@@ -75,9 +97,9 @@ Select 1:
 Identify the three places CSS can go, and rank them in terms of specificity:
 
 ```text
-1.
-2.
-3.
+1. Internally on the html sheet itself
+2. Externally: A style.css sheet
+3. Multiple external style sheets.
```

Comment:
Close!
1. The `style=` attribute, as in `<div style="color:red">`
2. Inline stylesheets using `<style>`
3. External stylesheets using `<link>`